style.css

```diff
@@ -322,28 +322,266 @@ textarea:focus {
 
 h2.get-started {
   padding-left: 0;
+  transition-duration: 0.5s;
+}
+
+.get-started p:hover {
+  transform: translate(20px);
 }
 
-.spining {
-  animation-name: spin;
-  animation-duration: 1s;
-  animation-iteration-count: 1;
+.desktop-only {
+  display: none;
 }
 
-@keyframes spin {
-  from {
-    transform: rotate(0);
+@media screen and (min-width: 992px) {
+  header {
+    flex-wrap: wrap;
+    flex-direction: row;
   }
 
-  to {
-    transform: rotate(360deg);
+  nav {
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+    height: 100px;
+    margin: 0;
+    border-bottom: 1px solid #dddfe6;
   }
-}
 
-.get-started p {
-  transition-duration: 1s;
-}
+  section.intro {
+    width: 40%;
+    padding-left: 5%;
+    padding-right: 5%;
+    height: 685px;
+    padding-top: 192px;
+    background-image: url("./img/image_dots.svg");
+    background-position: bottom right;
+    background-repeat: no-repeat;
+  }
 
-.get-started p:hover {
-  transform: translate(20px);
+  section.header-image {
+    width: 50%;
+    background-image: url("./img/image-geometry_desktop.svg");
+    height: 877px;
+  }
+
+  h1 {
+    font-size: 110px;
+    line-height: 110px;
+    height: 220px;
+    padding-bottom: 60px;
+  }
+
+  p.introduction {
+    padding-bottom: 60px;
+  }
+
+  button.primary-dark {
+    width: 254px;
+  }
+
+  .mobile-only {
+    display: none;
+  }
+
+  .desktop-only {
+    display: block;
+    margin-right: 50px;
+    font-family: Poppins, sans-serif;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: right;
+    color: #1a2236;
+    border-bottom: 2px solid #fff;
+  }
+
+  .desktop-only:hover {
+    border-bottom: 2px solid #1a2236;
+  }
+
+  main {
+    margin-top: 200px;
+    grid-template-columns: 11.1% 11.1% 11.1% 11.1% 11.1% 11.1% 11.1% 11.1% 11.1%;
+    grid-template-rows: 558px 80px 190px 80px 190px 275px 270px;
+  }
+
+  main h2 {
+    font-size: 110px;
+    transform: rotate(-90deg);
+    width: 11.1%;
+    height: 50px;
+    grid-column: 2 / 3;
+    position: relative;
+    top: 350px;
+  }
+
+  .item1 {
+    grid-column: 4 / 10;
+  }
+
```

Comment:
- [ ] Great job implementing media queries for your projects cards. The feature project looks great for 992px. However, for greater sizes the third project covers it. :wink:. 
- Maybe you can implement some more media queries for this card (between 992px and 1100px) (It's not a mandatory option you can find another way to fix it.)

![Feature project](https://user-images.githubusercontent.com/79876959/139561980-147654fe-6181-4eef-b31e-b5aa661d9cf6.PNG)